### PR TITLE
Add webui instructions to SLES-ES

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10805

### CODE REVIEW

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 1]
```diff
@@ -164,97 +164,49 @@ The parent channel will not contain any packages.
...
 ----
 
-. Proceed to Syncing Base media for {rhel} or {centos} 6,7, or 8.
-
-
-
-.**Procedure: Syncing Base Media**
+When the image is mounted, you can synchronize the base media.
+To manually synchronize the channels, navigate to menu:Software[Manage > Channels].
+Click each channel in the list, and:
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]: here you seem to start a new (pseudo) procedure.

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 2]
```diff
@@ -164,97 +164,49 @@ The parent channel will not contain any packages.
...
 
-The following steps require you to have mounted the base media for the selected distribution.
+. Navigate to the [guimenu]``Repositories`` tab.
+. Navigate to the [guimenu]``Sync`` subtab.
+. Click btn:[Sync Now] to begin synchronization immediately.
+. You can also create a synchronization schedule from this screen.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]: this looks "OPTIONAL:".

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 3]
```diff
@@ -236,17 +236,19 @@ endif::[]
 Synchronization can sometimes take several hours.
 ====
 
-When you have the channels set up, associate each channel with the appropriate repository, and synchronize then channels (optionally configure scheduled syncronization).
+When you have the channels set up, associate each channel with the appropriate repository, and synchronize the channels (optionally configure scheduled syncronization).
 
-To do this, proceed to menu:Software[Manage > Channels], and for each channel you created, click on it and:
+To do this, navigate to menu:Software[Manage > Channels], and for each channel you created, click it and:
 
 . Navigate to the [guimenu]``Repositories``
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]:
```suggestion
. Navigate to the [guimenu]``Repositories``.
```
Drive-by edit ;)

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 4]
```diff
@@ -181,12 +181,9 @@ mount -o loop /tmp/<iso_filename> /srv/www/htdocs/pub/rhel
 
 When the image is mounted, you can synchronize the base media.
 To manually synchronize the channels, navigate to menu:Software[Manage > Channels].
-Click each channel in the list, and:
-
-. Navigate to the [guimenu]``Repositories`` tab.
-. Navigate to the [guimenu]``Sync`` subtab.
-. Click btn:[Sync Now] to begin synchronization immediately.
-. You can also create a synchronization schedule from this screen.
+Click each channel in the list, and navigate to menu:[Repositories > Sync].
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]: now it's running text.  intentionally? ;)
```suggestion
+
* Click each channel in the list, and navigate to menu:[Repositories > Sync].
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 5]
```diff
@@ -181,12 +181,9 @@ mount -o loop /tmp/<iso_filename> /srv/www/htdocs/pub/rhel
...
+Click btn:[Sync Now] to begin synchronization immediately.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]:
```suggestion
* Click btn:[Sync Now] to begin synchronization immediately.
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 6]
```diff
@@ -181,12 +181,9 @@ mount -o loop /tmp/<iso_filename> /srv/www/htdocs/pub/rhel
...
+You can also create a synchronization schedule from this screen.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc]:
```suggestion
* You can also create a synchronization schedule from this screen.
```
Maybe, a simple itemized list would make sense?  If not, ignore my proposal.  It's not that important.

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 7]
```diff
@@ -238,11 +238,10 @@ Synchronization can sometimes take several hours.
 
 When you have the channels set up, associate each channel with the appropriate repository, and synchronize the channels (optionally configure scheduled syncronization).
 
-To do this, navigate to menu:Software[Manage > Channels], and for each channel you created, click it and:
-
-. Navigate to the [guimenu]``Repositories``
-. Navigate to the [guimenu]``Sync``and click btn:[Sync Now] to begin synchronization immediately.
-. You can also setup a scheduled synchronization from this screen.
+To manually synchronize the channels, navigate to menu:Software[Manage > Channels].
+Click each channel in the list, and navigate to menu:[Repositories > Sync].
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: If you accept the proposal, we need it here again.

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-sleses.adoc]: Yeah, it didn't seem worth making it into a procedure, especially since it's in a couple of different places.

File: modules/client-configuration/pages/clients-sleses.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-sleses.adoc]: It's got to be a numbered list, because the order is important. So we either turn it into prose, or we make it a proper procedure. Making it a procedure seem like overkill to me, but I could be persuaded :wink:

Comment (keichwa) [reply modules/client-configuration/pages/clients-sleses.adoc]: Thinking about it again, I'd rather vote to shorten it and turn it into prose.  I hope I edited my previous comment correctly...

**Comment by keichwa** [comment]
On 2020-03-18 04:51, Lana Brindley wrote:

> It's got to be a numbered list, because the order is important. So we
> either turn it into prose, or we make it a proper procedure. Making it
> a procedure seem like overkill to me, but I could be persuaded :wink:

Ok, make sense.  I'd just delete this additional info:

> +You can also create a synchronization schedule from this screen.

If the user needs info on this, he can click the question mark and read
the Reference.

-- 
Karl Eichwalder

**Comment by calancha** [comment]
At SUSE/spacewalk#10805 it was mentioned to have both options:
- synch by GUI.
- synch by command line.
This PR has replaced the command line description by GUI one; I don't see a discussion on that decision.  Do we really want to show just the GUI way?

**Comment by Loquacity** [comment]
> At [SUSE/spacewalk#10805](https://github.com/SUSE/spacewalk/issues/10805) it was mentioned to have both options:
> 
> * synch by GUI.
> * synch by command line.
>   This PR has replaced the command line description by GUI one; I don't see a discussion on that decision.  Do we really want to show just the GUI way?

In most of the documentation we only use the GUI method (there are exceptions, of course).
